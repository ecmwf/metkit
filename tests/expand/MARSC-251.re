retrieve,
    class=ti,
    date=-3,
    time=12,
    origin=all,
    expver=all,
    type=cf,
    stream=enfo,
    levtype=sfc,
    param=2t,
    step=24,
    expect=any,
    target=data.reference 