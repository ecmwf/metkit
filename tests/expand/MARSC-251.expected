RETRIEVE,
    CLASS      = TI,
    TYPE       = CF,
    STREAM     = ENFO,
    EXPVER     = ALL,
    REPRES     = SH,
    LEVTYPE    = SFC,
    PARAM      = 167,
    DATE       = 20250414,
    TIME       = 1200,
    STEP       = 24,
    ORIGIN     = ALL,
    TARGET     = data.reference,
    PADDING    = 0,
    PROCESS    = LOCAL,
    EXPECT     = ANY,
    NUMBER     = 0